cookbook/common/leaf.mk

```diff
@@ -45,14 +45,24 @@ endif
 # The Flyte project that we want to register under
 export PROJECT ?= flytesnacks
 
+PATH_TO_EXAMPLE = $(shell echo "${CURDIR}" | grep -o "cookbook.*")
+
 # If the REGISTRY environment variable has been set, that means the image name will not just be tagged as
 #   flytecookbook:<sha> but rather,
 #   ghcr.io/flyteorg/flytecookbook:<sha> or whatever your REGISTRY is.
 ifdef REGISTRY
 	FULL_IMAGE_NAME = ${REGISTRY}/${IMAGE_NAME}
+	EXAMPLE_OUTPUT_DIR = ${CURDIR}/_pb_output
+    COMMAND = docker
+    BUILD_CONTEXT = ${CURDIR}/..
+    DOCKER_FILE = Dockerfile
 endif
 ifndef REGISTRY
 	FULL_IMAGE_NAME = ${IMAGE_NAME}
+	EXAMPLE_OUTPUT_DIR = /root/${PATH_TO_EXAMPLE}/_pb_output
+	COMMAND = flytectl sandbox exec -- docker
```

Comment:
REGISTRY or no REGISTRY shouldn't determine whether we use sandbox or not... that should be based off maybe the `FLYTE_HOST` value... if it's localhost, use sandbox command... etc.

Reply:
@EngHabu currently I just replicated what we have today. In docker push, we have a condition on the registry. I didn't use flytehost for the condition because flytectl is reading config from there home directory/Env, and i didn't think it's a required parameter for flytesnacks but the registry is  required parameter